recipes/libogg/meta.yaml

```diff
@@ -0,0 +1,35 @@
+{% set version = "1.3.2" %}
+
+package:
+    name: libogg
+    version: {{ version }}
+
+source:
+    fn: libogg-{{ version }}.tar.gz
+    url: http://downloads.xiph.org/releases/ogg/libogg-{{ version }}.tar.gz
+    sha256: e19ee34711d7af328cb26287f4137e70630e7261b17cbe3cd41011d73a654692
+
+build:
+    number: 0
+    skip: True  # [win]
+
+requirements:
+    build:
+        - gcc
+
+test:
+    commands:
+        - test -f ${PREFIX}/lib/libogg.a
+        - test -f ${PREFIX}/lib/libogg.la
+        - test -f ${PREFIX}/lib/libogg.dylib
```

Comment:
We might need a selector as Linux and OS X use different extensions. This should work.

```
- test -f ${PREFIX}/lib/libogg.so     # [linux]
- test -f ${PREFIX}/lib/libogg.dylib  # [osx]
```